docs/api.md

```diff
@@ -87,3 +87,14 @@ System.set('http://site.com/normalized/module/name.js', {
 `module` is an object of names to set as the named exports.
 
 If `module` is an existing Module Namespace, it will be used by reference.
+
+#### System[Symbol.iterator]() -> Iterator
```

Comment:
Can we make this `System.entries()` rather, and then say "also available at `System[Symbol.iterator]`.